site/docs/main/configuration.md

```diff
@@ -1,9 +1,55 @@
 # Contour Configuration Reference
 
+- [Serve Flags](#serve-flags)
 - [Configuration File](#configuration-file)
 - [Environment Variables](#environment-variables)
 - [Bootstrap Config File](#bootstrap-config-file)
 
+## Serve Flags
+
+The `contour serve` command is the main command which is used to watch for Kubernetes resource and process them into Envoy configuration which is then streamed to any Envoy via its xDS gRPC connection.
+There are a number of flags that can be passed to this command which further configures how Contour operates. 
+Many of these flags are mirrored in the [Contour Configuration File](#configuration-file).
```

Comment:
Worth adding a note about the precedence order of CLI flags vs. the config file?

Reply:
yes, good call!

Reply:
I added a `Configuration` section at the top to explain the ordering.